displayer.go

```diff
@@ -0,0 +1,21 @@
+package drivers
+
+import "image/color"
+
+type Displayer interface {
+	// Size returns the current size of the display.
+	Size() (x, y int16)
+
+	// SetPizel modifies the internal buffer.
+	SetPixel(x, y int16, c color.RGBA)
```

Comment:
This should also return `error`.

Reply:
I would not return an error here: the only possible error is an index out of bounds which is a programming error. In my opinion, things that are clearly a bug in the program should just panic (just like an index out of bounds). If you want to know more, read about "abandonment" in http://joeduffyblog.com/2016/02/07/the-error-model/.